examples/playground/app.tsx

```diff
@@ -132,7 +132,7 @@ type ProductItemProps = {
 
 function ProductItem({ hit, insights, components }: ProductItemProps) {
   return (
-    <Fragment>
+    <a href="#" className="aa-ItemLink">
```

Comment:
We can use the actual link.

```suggestion
    <a href={hit.url} className="aa-ItemLink">
```